invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/frontpage/override.js

```diff
@@ -0,0 +1,7 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2022 CERN.
+//
+// Invenio App RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+export const overriddenComponents = {};
```

Comment:
Instead of creating another wrapper for `RecordsListItem`, can we not just use the default `ListItem` from `react-searchkit` in the frontpage `RecordsList`, and then override it with the original `RDMRecordResultsListItem` in this file, and that way keep it as it was in the search components?